jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -0,0 +1,200 @@
+- job:
+    name: kubernetes-pull-test-unit-integration
+    logrotate:
+        daysToKeep: 7
+    properties:
+        - github:
+            url: https://github.com/kubernetes/kubernetes
+        - throttle:
+            max-per-node: 2
+            option: project
+        - raw:
+            xml: |
+                <com.cloudbees.plugins.JobPrerequisites plugin="slave-prerequisites@1.0">
+                    <script>docker version gcloud version</script>
+                    <interpreter>shell script</interpreter>
+                </com.cloudbees.plugins.JobPrerequisites>
+    parameters:
+        - string:
+            name: sha1
+            description: 'sha1 or refname (e.g. origin/pr/N/head) to build'
```

Comment:
we can probably remove this (I don't think we ever use it), but let's do that in a follow-up.